docs/_data/table/en/security-configuration.yml

```diff
@@ -1,6 +1,6 @@
 alluxio.security.authentication.type:
   "The authentication mode. Currently three modes are supported: NOSASL, SIMPLE,
-  CUSTOM. The default value NOSASL indicates that authentication is not enabled."
+  CUSTOM. The default value SIMPLE indicates that authentication is enabled."
```

Comment:
+1 for more explanation here. Basically, SIMPLE means server believes whoever the client claims it is.